### ISSUE TITLE
Canvas click not firing when item is selected

### ISSUE THREAD

**Issue by florianbepunkt** [namespace-ee/react-calendar-timeline#312]
Not sure if this is a bug or by design

I have a timeline and specify 

```
onCanvasClick={(e) => console.log('canvas clicked')}
onItemSelect={(e) => console.log('item selected')}
```

When I select an Item and item is displayed as being selected, I need to click two times on the canvas to fire the event. The first click on canvas deselects the item, but does not fire the event, the second click fire the onCanvasClick event.

Is this a bug? I would expect onCanvas click to fire the first time

EDIT: Here is a super simple repro based on your template, just added the props mentioned above

**Comment by mcMickJuice** [comment]
Is this issue occurring in the base example (the link that is in the issue template)?

Any reason you didn't use the issue template for this issue? 😀

**Comment by florianbepunkt** [comment]
Sorry, my bad…  here we go: https://codesandbox.io/s/l211p68z6m

**Comment by mcMickJuice** [comment]
Yea, that's a bug. Thanks for posting the code!

**Comment by mcMickJuice** [comment]
Should be fixed in next release

**Comment by mcMickJuice** [comment]
released with 0.16.2
